plugins/community/roadrunner-server-php-grpc/v4.2.0/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
+FROM golang:1.20.6-bullseye AS build
+RUN --mount=type=cache,target=/go/pkg/mod \
+    git clone --depth=1 --branch=v4.2.0 https://github.com/roadrunner-server/grpc.git \
+    && cd grpc/protoc_plugins/protoc-gen-php-grpc \
+    && CGO_ENABLED=0 go build -ldflags "-s -w -X github.com/roadrunner-server/roadrunner/v2/internal/meta.version=v4.2.0" -trimpath
```

Comment:
I don't know if this linker symbol is required, but I saw it in the Dockerfile that used to build the buf plugin.

Reply:
`gRPC` protoc plugin does not have its own version. Previously it was versioned according to the RoadRunner versions (like `v2023.1.0).
`-s -w` should be used separately. `-s` or `-w`, because `-s` remove symbol table with debug symbols, while `-w` removes only debug symbols, so their purpose overlaps. I prefer using `-s` for the release builds.

Reply:
But, I'm fine to tag plugin with the `gRPC` plugin (roadrunner-server/grpc) version, like `v4.2.0`, thank you for the super fast update 👍🏻

Reply:
@rustatian - So should I remove the version symbol? I didn't see it used in the generated code - I'm guessing it is for a usage statement and not needed for a remote plugin?

Reply:
Confirmed that it is used in the CLI / server in roadrunner and not referenced by the plugin, so I removed the custom linker symbol.

Reply:
Yeah, right, it is used in the main RR repository, safe to delete, thanks 👍🏻